java/client/test/org/openqa/selenium/print/BUILD.bazel

```diff
@@ -0,0 +1,15 @@
+load("@rules_jvm_external//:defs.bzl", "artifact")
+load("//java:defs.bzl", "java_binary", "java_test_suite")
+
+java_test_suite(
+    name = "SmallTests",
+    size = "small",
+    srcs = glob(["*Test.java", "print/*Test.java"]),
```

Comment:
if you add `print/*Test.java` to `SMALL_TESTS` in `java/client/test/org/openqa/selenium/BUILD.bazel` you could get rid of this file and things will start working